analyzers/tests/SonarAnalyzer.UnitTest/PackagingTests/GeneratedResourcesTest.cs

```diff
@@ -32,7 +32,7 @@ public class GeneratedResourcesTest
 
         static GeneratedResourcesTest()
         {
-            RspecFolderPath = Path.GetFullPath(Path.Combine(PathResolver.RelativePathToTestProjectRoot, @"..\..\rspec\"));
+            RspecFolderPath = Path.GetFullPath(Path.Combine(Paths.TestProjectRoot, @"..\..\rspec\"));
```

Comment:
Actually `..\..\` (or `"..", ".."`) appears 3 times. What about adding also `Paths.AnalyzersRoot`?

Reply:
It makes sense. I will add it.